piker/brokers/ib.py

```diff
@@ -811,78 +777,72 @@ def normalize(
     return data
 
 
-_local_buffer_writers = {}
-
-
-@asynccontextmanager
-async def activate_writer(key: str) -> (bool, trio.Nursery):
-    """Mark the current actor with module var determining
-    whether an existing shm writer task is already active.
-
-    This avoids more then one writer resulting in data
-    clobbering.
-    """
-    global _local_buffer_writers
-
-    try:
-        assert not _local_buffer_writers.get(key, False)
-
-        _local_buffer_writers[key] = True
-
-        async with trio.open_nursery() as n:
-            yield n
-    finally:
-        _local_buffer_writers.pop(key, None)
-
-
-async def fill_bars(
+async def backfill_bars(
     sym: str,
-    first_bars: list,
-    shm: 'ShmArray',  # type: ignore # noqa
+    shm: ShmArray,  # type: ignore # noqa
     # count: int = 20,  # NOTE: any more and we'll overrun underlying buffer
-    count: int = 6,  # NOTE: any more and we'll overrun the underlying buffer
+    count: int = 10,  # NOTE: any more and we'll overrun the underlying buffer
```

Comment:
iirc this was < 6 to avoid tws throttling on another system I tried.

We might want to include this change as a default?